restaurants/restModel.js

```diff
@@ -0,0 +1,24 @@
+const db = require("../database/db-config");
+
+module.exports = {
+    // get,
+    getById,
+    // getUserPosts,
+    insert,
+    // update,
+    // remove,
+  };
+
+  function getById(restaurant_id) {
+    return db('restaurants')
+      .where({ restaurant_id })
+      .first();
+  }
+
+  function insert(restaurant) {
```

Comment:
please take a look at how the insert for the register was, all the helper functions for insert and update should be returning something